plutus-playground-client/src/MainFrame.purs

```diff
@@ -389,7 +390,15 @@ eval (EvaluateActions next) = do
           assign _evaluationResult Loading
           result <- lift $ postEvaluation evaluation
           assign _evaluationResult result
-          replaceViewOnSuccess result Simulations Transactions
+          -- If we got a successful result, switch tab.
+          case result of
+            Success (JsonEither (Left _)) -> pure unit
+            _ -> replaceViewOnSuccess result Simulations Transactions
+          -- Update the error display.
+          -- editorSetAnnotations
```

Comment:
We're no longer setting the error annotations? Is that right?

Reply:
We weren't before, this is my WIP code, I will remove it. The errors are set to display elsewhere but there are now different types I think so we can't set annotations (IIRC).

Reply:
removed